main.go

```diff
@@ -149,22 +149,32 @@ func handleEvent(event vtypes.BaseEvent, m *event.Manager) (string, string, erro
 
 	topic := convertToTopic(eventType)
 
-	message, _ := json.Marshal(OutboundEvent{
+	message, err := json.Marshal(OutboundEvent{
 		Topic:       topic,
 		Category:    category,
-		CreatedTime: gotEvent.CreatedTime,
 		UserName:    gotEvent.UserName,
+		CreatedTime: gotEvent.CreatedTime,
+		MoRef:       ref,
 	})
+	if err != nil {
+		return "", "", errors.Wrap(err, "error marshaling outboundevent")
+	}
+
+	log.Println(string(message))
```

Comment:
Can we do the print one level up from here and only if a verbose env-var is set?

Reply:
Which format do you prefer for the env, e.g. `VCVERBOSE`?

Reply:
will fix in a follow-up PR if ok